### PR TITLE
add version CLI command

### PR DESCRIPTION
**Pull request by edjroz**
Solves #693

### CODE REVIEW

File: app/cmd/cli/query.go [hunk 1]
```diff
@@ -94,6 +96,14 @@ var queryHeight = &cobra.Command{
 		fmt.Printf("Block Height: %d\n", res)
 	},
 }
+var version = &cobra.Command{
```
Comment (oten91) [inline app/cmd/cli/query.go]: Command is under query not root 

```
oten91@OMP % ./pocket_core version
Error: unknown command "version" for "pocket"
Run 'pocket --help' for usage.
unknown command "version" for "pocket"
oten91@OMP pocket_core % ./pocket_core query version
Version: RC-0.2.0
oten91@OMP pocket_core % 
```

is this intended?

File: app/cmd/rpc/server.go [hunk 2]
```diff
@@ -2,17 +2,18 @@ package rpc
...
-	"github.com/julienschmidt/httprouter"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"net/http"
-)
 
-const (
-	APIVersion = "0.2.0"
```
Comment (oten91) [inline app/cmd/rpc/server.go:13]: what happens to releases, do we change the app.Tag value to "" each time?

**Review by oten91 — CHANGES_REQUESTED** [review]

File: app/cmd/cli/query.go (hunk 1 above)
Comment (edjroz) [reply app/cmd/cli/query.go]: It is intended, however we can move this to root if everyone agrees.

File: app/cmd/rpc/server.go (hunk 2 above)
Comment (edjroz) [reply app/cmd/rpc/server.go:13]: We change the version as needed, and the tag on a git tag basis

Comment (oten91) [reply app/cmd/rpc/server.go:13]: what do you think about merging app.Tag and app.Version in 1 variable?

Comment (edjroz) [reply app/cmd/rpc/server.go:13]: the usage of `app.Tag` allows for fine grained behavior for whenever we release `stable` tags which should be separate of `RC`

**Review by oten91 — APPROVED** [review]

**Pull request merged**